integrations/aws/aws-ec2-monitoring/sensu-integration.yaml

```diff
@@ -0,0 +1,384 @@
+        description: >-
+          How do you want to process metrics collected by this integration?
+        ref: core/v2/pipeline/metadata/name
+        refFilter: .labels.provider == "metrics"
+  resource_patches:
+    - resource:
+        api_version: core/v2
+        type: CheckConfig
+        name: aws-ec2-metrics
+      patches:
+        - op: replace
+          path: /metadata/name
+          value: "aws-[[aws_region]]-ec2-metrics"
+        - op: replace
+          path: /spec/command
+          value: >-
+            sensu-cloudwatch-check
+            --preset EC2
+            --region {{ or .annotations.aws_region "[[aws_region]]" }}
+            --dimension-filters InstanceId={{ or .annotations.aws_ec2_instance_id .annotations.instance_id .name }}
+            --period-minutes 10
+        - op: add
+          path: /spec/output_metric_tags/-
+          value:
+            name: Region
+            value: "[[aws_region]]"
+        - op: replace
+          path: /spec/subscriptions
+          value: subscriptions
+        - op: add
+          path: /spec/subscriptions/-
+          value: "aws/[[aws_region]]"
+        - op: replace
+          path: /spec/output_metric_thresholds/0/thresholds/0/max
+          value: "[[cpu_utilization_warning]]"
+        - op: replace
+          path: /spec/output_metric_thresholds/0/thresholds/1/max
+          value: "[[cpu_utilization_critical]]"
+        - op: add
+          path: /spec/env_vars/-
+          value: "AWS_PROFILE=[[aws_profile.env_var]]"
+        - op: add
+          path: /spec/env_vars/-
+          value: "AWS_ACCESS_KEY_ID=[[aws_access_key_id.env_var]]"
+        - op: add
+          path: /spec/env_vars/-
+          value: "AWS_SECRET_ACCESS_KEY=[[aws_secret_access_key.env_var]]"
+        - op: add
+          path: /spec/secrets/-
+          value:
+            secret: "[[aws_profile.secret]]"
+            name: AWS_PROFILE
+        - op: add
+          path: /spec/secrets/-
+          value:
+            secret: "[[aws_access_key_id.secret]]"
+            name: AWS_ACCESS_KEY_ID
+        - op: add
+          path: /spec/secrets/-
+          value:
+            secret: "[[aws_secret_access_key.secret]]"
+            name: AWS_SECRET_ACCESS_KEY
+        - op: add
+          path: /spec/pipelines/-
+          value:
+            api_version: core/v2
+            type: Pipeline
+            name: "[[alert_pipeline]]"
+        - op: add
+          path: /spec/pipelines/-
+          value:
+            api_version: core/v2
+            type: Pipeline
+            name: "[[incident_pipeline]]"
+        - op: add
+          path: /spec/pipelines/-
+          value:
+            api_version: core/v2
+            type: Pipeline
+            name: "[[metrics_pipeline]]"
+  post_install:
+    - type: section
+      title: Success
+    - type: markdown
+      body: |
+        The AWS EC2 monitoring integration was successfully installed for the `[[aws_region]]` AWS Region.
```

Comment:
Is there benefit to add a post install section reminding people to set the instance id annotation?

Reply:
Yes, doing a review of the `post_install` content on all integrations now. FINAL PASS